### PR TITLE
Starts on this weeks hw MTA

### PR DESCRIPTION
**Pull request by RaengC**
_If you had no problems you can just leave it empty_

> How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
8, a bit sick so struggling this week
> Was there anything that you struggled with?

> Is there any particular code you want me to look into?
stared where I'm currently stuck
> Is there anything that you'd like some further information on?

> Roughly how long did it take?
1-2 hours

### CODE REVIEW

**Comment by RaengC** [comment]
Books

How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible
7, 
Was there anything that you struggled with?
its displaying the wrong book title! unsure why??
Is there any particular code you want me to look into?
above
Is there anything that you'd like some further information on?
what I did wrong!
Roughly how long did it take?
about 2 hours

File: Raeng Castagna/week_4/mta.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# p start_line
+print "Enter start station: "
+start_station = gets.chomp.to_s
+# p start_station
+print "Enter end line: "
+end_line = gets.chomp.to_sym
+print "Enter end stop: "
+end_station = gets.chomp.to_s
+
+case start
```
Comment (CamillaCdC) [inline Raeng Castagna/week_4/mta.rb:38]: undefined variable

File: Raeng Castagna/week_4/books/main.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+require 'pry'
+require 'sinatra'
+require 'sinatra/reloader'
+require 'httparty' # add party to require google books api link
+
+book_url = "https://www.googleapis.com/books/v1/volumes?q=title:#{ @book_title }"
```
Comment (CamillaCdC) [inline Raeng Castagna/week_4/books/main.rb:6]: by setting the book_url outside of the method you're searching for the same book every time. hence displaying the same book every time you search.

File: Raeng Castagna/week_4/books/main.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+get '/' do
+  erb :home
+end
+
+get '/title' do
+  @book_title = params[:book_title]
+
+  @info = HTTParty.get book_url;
+
+  @image = @info["items"][0]["volumeInfo"]["imageLinks"]["thumbnail"]
```
Comment (CamillaCdC) [inline Raeng Castagna/week_4/books/main.rb:17]: this is right.

File: Raeng Castagna/week_4/books/main.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+require 'httparty' # add party to require google books api link
+
+book_url = "https://www.googleapis.com/books/v1/volumes?q=title:#{ @book_title }"
+
+get '/' do
+  erb :home
+end
+
+get '/title' do
+  @book_title = params[:book_title]
```
Comment (CamillaCdC) [inline Raeng Castagna/week_4/books/main.rb:13]: this is right

File: Raeng Castagna/week_4/books/main.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+  @info = HTTParty.get book_url;
```
Comment (CamillaCdC) [inline Raeng Castagna/week_4/books/main.rb:15]: This is the line you need to fix.

**Comment by CamillaCdC** [comment]
MTA: 
You've got an undefined variable start in your case statement, which I think as a result is stuffing up your loop. Try fixing that and see if you can get your loop working. Happy to take another look at it if you want to re-submit. 

BOOKS: 
You're displaying the same image for every book because you're performing the same search because you've defined book_url outside of the method. You want to perform this HTTParty.get search every time the function is called with the new @book_title variable.

**Pull request merged**